wire-ios-sync-engine/Source/SessionManager/SessionFactories.swift

```diff
@@ -60,6 +60,7 @@ open class AuthenticatedSessionFactory {
         for account: Account,
         coreDataStack: CoreDataStack,
         configuration: ZMUserSession.Configuration,
+        analyticsSessionConfiguration: AnalyticsSessionConfiguration? = nil,
```

Comment:
Is this still needed?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e12bacb446dc277c68c1c3efba1ebb121183ea68